packages/core/src/lib/icon/icons.ts

```diff
@@ -77,6 +77,28 @@ export const paths = {
   'robot-outline': MDI.mdiRobotOutline,
   domain: MDI.mdiDomain,
   'database-outline': MDI.mdiDatabaseOutline,
+  'part-awaiting-setup':
```

Comment:
Should we distinguish the custom icon names by prepending them with viam or custom?

Reply:
good call, updated to prepend custom icons with "viam"